### PR TITLE
Define primitve restart values

### LINKED ISSUE #1002: Specify primitive restart behavior
The WebGPU spec does not fully describe the primitive restart behavior (ie. always on + magic value). 

While obvious to some, it would be worth clarifying what our standard is.

### PR DESCRIPTION
**Pull request by toji**
Fixes #1002

***
<a href="https://pr-preview.s3.amazonaws.com/toji/gpuweb/pull/1025.html" title="Last updated on Aug 25, 2020, 4:40 PM UTC (fe7cff2)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/gpuweb/gpuweb/1025/415fc87...toji:fe7cff2.html" title="Last updated on Aug 25, 2020, 4:40 PM UTC (fe7cff2)">Diff</a>

### CODE REVIEW

File: spec/index.bs [hunk 1]
```diff
@@ -3703,6 +3703,36 @@ enum GPUIndexFormat {
 };
 </script>
 
+The index format determines both the data type of index values in a buffer and, when used with
+strip primitive topologies ({{GPUPrimitiveTopology/"line-strip"}} or
+{{GPUPrimitiveTopology/"triangle-strip"}}) also specifies the primitive restart value. The
+<dfn dfn>primitive restart value</dfn> indicates which index value indicates that a new primitive
+should be started rather than continuing to construct the triangle strip with the prior two indexed
```
Comment (kvark) [inline spec/index.bs:3710]: I'm not sure if it's necessarily "two" indexed vertices. I'd expect it also work if there is "one" remaining vertex.

File: spec/index.bs [hunk 2]
```diff
@@ -3703,6 +3703,36 @@ enum GPUIndexFormat {
...
+vertices.
+
+{{GPURenderPipelineDescriptor}}s that specify a strip primitive topology must not leave the
```
Comment (kvark) [inline spec/index.bs:3713]: maybe "must not have" instead of "must not leave"?

**Comment by kvark** [comment]
@toji do you want to address the 2 nits I noted here? I'm fine with merging otherwise.

**Comment by toji** [comment]
I'll fix them, thanks!

File: spec/index.bs (hunk 1 above)
Comment (toji) [reply spec/index.bs]: Fixed!

File: spec/index.bs (hunk 2 above)
Comment (toji) [reply spec/index.bs]: Fixed!

**Pull request merged**